src/service/repo/anonymousMessageRepo.ts

```diff
@@ -8,6 +8,7 @@ type SendPayloadType = {
   message: string;
   messageType?: string;
   replyMessageId?: string;
+  attachments?: any;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachments` property is of type `any`. It's generally a good practice to avoid using `any` as it can lead to runtime errors. Consider defining an interface for the attachments if possible.
```diff
-  attachments?: any;
+  attachments?: AttachmentType[];
```
Where `AttachmentType` could be something like:
```typescript
interface AttachmentType {
  name: string;
  type: string;
  data: Buffer | string; // or whatever type you expect here
}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->